commit 69c7fe5ff35bca7a55f9ecec73ef9fa5f5bdd9a1
Author: John Johansen <john.johansen@canonical.com>
Date:   Tue Aug 9 13:47:43 2016 -0700

    apparmor: add a stack_version file to allow detection of bug fixes
    
    Signed-off-by: John Johansen <john.johansen@canonical.com>
    Signed-off-by: Leann Ogasawara <leann.ogasawara@canonical.com>

diff --git a/security/apparmor/apparmorfs.c b/security/apparmor/apparmorfs.c
index d937fd7..8d45829 100644
--- a/security/apparmor/apparmorfs.c
+++ b/security/apparmor/apparmorfs.c
@@ -1034,6 +1034,7 @@ static struct aa_fs_entry aa_fs_entry_domain[] = {
 	AA_FS_FILE_BOOLEAN("change_onexec",	1),
 	AA_FS_FILE_BOOLEAN("change_profile",	1),
 	AA_FS_FILE_BOOLEAN("stack",		1),
+	AA_FS_FILE_STRING("version", "1.1"),
 	{ }
 };
 
